recipe/meta.yaml

```diff
@@ -20,9 +20,11 @@ requirements:
     - {{ compiler('cxx') }}
   host:
     - hdf5
+    - hdf5-static
```

Comment:
We should be able to use only `hdf5-static` but we need to add that to the pinning scheme first for the re-render to work properly.

Reply:
Ok sounds good